src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,53 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100, hidden = false) {
+    super(name);
```

Comment:
The `super(name)` call should also pass the `health` parameter to the parent `Animal` class constructor. It should be `super(name, health);` to ensure the `health` property is correctly initialized in the `Animal` class.